draft-ietf-dnsop-svcb-https.md

```diff
@@ -1625,8 +1628,9 @@ although the output length in this document is not limited to 255 octets.
 ## Decoding a comma-separated list {#value-list}
 
 In order to represent lists of items in zone files, this specification uses
-comma-separated lists.  When "," is not escaped (by a preceding "\\"), it
-separates items in the list.  (For simplicity, empty items are not allowed.)
+comma-separated lists.  When the allowed items in the list cannot contain ","
+or "\\", this is trivial.  (For simplicity, empty items are not allowed.)
```

Comment:
```suggestion
or "\\", this is trivial.  If items in the list are allowed to contain ",",
it can be specified by escaping as "\\,"  or by relying on the 
relying on the opaque key format (e.g. `key1=\002h2`) for 
implementations that do not support escaping.
(For simplicity, empty items are not allowed.)
```

It seems worth including an example here for here for potential zone file authors, not just implementors?

Reply:
I think it's OK to mention the opaque key format here (although I doubt many zone file authors will be reading the appendix), but I would prefer not to duplicate the example.

Reply:
I expanded the text here in a different way.  Please review.